pkg/openthread/contrib/platform_uart.c

```diff
@@ -13,34 +13,148 @@
  * @brief       Implementation of OpenThread UART platform abstraction
  *
  * @author      Jose Ignacio Alamos <jialamos@uc.cl>
+ * @author      Baptiste Clenet <bapclenet@gmail.com>
  * @}
  */
 
 #include <stdint.h>
 #include <stdio.h>
 
 #include "periph/uart.h"
+#include "openthread/types.h"
 #include "openthread/platform/uart.h"
+#include "ot.h"
+
+#define ENABLE_DEBUG (0)
+#include "debug.h"
+
+#define UART_OPENTHREAD_BAUDRATE            (115200U)
+#define UART_OPENTHREAD                     UART_DEV(0)
+#define OPENTHREAD_SPINEL_FRAME_MARKER      0x7e
+
+serial_msg_t * gSerialMessage[OPENTHREAD_NUMBER_OF_SERIAL_BUFFER];
+uint8_t gSerialBuff[OPENTHREAD_NUMBER_OF_SERIAL_BUFFER][OPENTHREAD_SERIAL_BUFFER_SIZE];
+
+#ifdef MODULE_LIBOPENTHREAD_NCP_FTD
+int8_t getFirstEmptySerialBuffer(void) {
+    uint8_t i =0;
```

Comment:
Ok to move but why use unsignerd, the number of buffer will never goes over 255 so I keep uint8_t